axelor-redmine/src/main/java/com/axelor/apps/redmine/imports/service/projects/RedmineImportProjectServiceImpl.java

```diff
@@ -320,7 +334,15 @@ public void setProjectFields(
     project.setName(redmineProject.getName());
     project.setDescription(getHtmlFromTextile(redmineProject.getDescription()));
     project.setCompany(companyRepo.find(defaultCompanyId));
-
+    projectStatusRepo
```

Comment:
i think it is better to perform below logic only when project id is not set means project is new
as in existing projects it is possible that user had intentionally keep this sets empty or selected records as per their requirements